src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBuffer.cs

```diff
@@ -499,7 +499,7 @@ internal string String
         }
 
         // use static list of format strings indexed by scale for perf
-        private static readonly string[] s_katmaiDateTimeOffsetFormatByScale = new string[] {
+        private static readonly string[] s_2008DateTimeOffsetFormatByScale = new string[] {
```

Comment:
Could I ask you to have the version with `sql` prefix wherever it's meaningful like here?
```suggestion
        private static readonly string[] s_sql2008DateTimeOffsetFormatByScale = new string[] {
```

Reply:
Done.